### ISSUE TITLE
Possible bug in insulation?

### ISSUE THREAD

**Issue by gdolsten** [open2c/cooltools#523]
in api.py, annotate, should this line:

```
        anns.append(
            ann2.iloc[bin2 - bmin]
            .rename(columns=lambda x: x + "2")
            .reset_index(drop=True)
        )

```

be:

```

        anns.append(
            ann2.loc[bin2 - bmin]
            .rename(columns=lambda x: x + "2")
            .reset_index(drop=True)
        )

```

?

**Comment by Phlya** [comment]
https://github.com/open2c/cooler/pull/422 is this what you mean? In cooler.annotate?

**Comment by gdolsten** [comment]
Yes, sorry. I just realized that the bug was in cooler and not in cooltools. On my current install, calling insulation scores is broken.

**Comment by Phlya** [comment]
Ok, thank you for reporting. As a temporary fix, you can downgrade cooler to 0.9.3
